apps/web/src/design-system/popover/Popover.tsx

```diff
@@ -0,0 +1,76 @@
+import { MouseEventHandler, ReactNode } from 'react';
+import { Popover as MantinePopover, PopoverProps as MantinePopoverProps, useMantineTheme } from '@mantine/core';
+
+import { colors, shadows } from '../config';
+import { Label } from '../typography/label';
+
+type PopoverProps = {
+  target: ReactNode;
+  content?: ReactNode;
+  title?: string;
+  description?: string;
+  url?: string;
+  urlText?: string;
+  onUrlClick?: MouseEventHandler<HTMLAnchorElement>;
+  titleGradient: 'red' | 'blue' | 'none';
+} & Omit<MantinePopoverProps, 'children'>;
+
+export const Popover = ({
```

Comment:
moved this code from the `NodeStep` popovers shown on the Demo Workflow Tree to the design system as we will use similar popovers in other places